src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -62,25 +63,27 @@ public static int Main(string[] args)
                     "-s|--source <source>",
                     Strings.Restore_Switch_Source_Description,
                     CommandOptionType.MultipleValue);
+
                 var packagesDirectory = restore.Option(
                     "--packages <packagesDirectory>",
                     Strings.Restore_Switch_Packages_Description,
                     CommandOptionType.SingleValue);
-                var parallel = restore.Option(
-                    "-p|--parallel <noneOrNumberOfParallelTasks>",
-                    string.Format(
-                        CultureInfo.CurrentCulture,
-                        Strings.Restore_Switch_Parallel_Description,
-                        RestoreRequest.DefaultDegreeOfConcurrency),
-                    CommandOptionType.SingleValue);
+
+                var disableParallel = restore.Option(
```

Comment:
Originally I kept both flags, but it was very confusing to allow parallel projects on one switch while setting the number of threads per project on another switch.

MSBuild has BuildInParallel=true along with MaxCPUCount, but if the cpu count is 1 then BuildInParallel is turned off. In the nuget scenario here the two are independent which makes it hard to figure out the best settings. Also, this is also running all async and not using Task.Run to force the projects onto different threads, so .NET is doing a lot of the work here.

What we have is basically the best settings by default, and then --disable-parallel is "safe mode"